Cargo.toml

```diff
@@ -8,3 +8,10 @@ edition = "2021"
 [dependencies]
 clap = { version = "4.3.8", features = ["derive"] }
 lazy-regex = "2.5.0"
+rustyline = "=5.0.2"
+dialoguer = "0.11.0"
+
+[profile.dev]
+opt-level = "z"
```

Comment:
Why are you choosing to optimize for binary size rather than say performance or comp times?

Reply:
This software is already fast, and the file size was big. So I think is better to sacrifice some optmization in exchange for a small file. And, about the compilation time, well this is a process that doesn't need to be done all the time, and some people can even compile it just once, so, again, is for the best experience in my opinion.